cmd/cli/app/history/history_list.go

```diff
@@ -116,11 +118,20 @@ func listCommand(ctx context.Context, cmd *cobra.Command, _ []string, conn *grpc
 		req.To = timestamppb.New(to)
 	}
 
-	resp, err := client.ListEvaluationHistory(ctx, req)
+	rID := uuid.New().String()
+	md := metadata.Pairs("request-id", rID)
+	ctx = metadata.NewOutgoingContext(ctx, md)
+
+	var header, trailer metadata.MD
+	resp, err := client.ListEvaluationHistory(ctx, req, grpc.Header(&header), grpc.Trailer(&trailer))
 	if err != nil {
 		return cli.MessageAndError("Error getting profile status", err)
 	}
 
+	fmt.Printf("%+v\n", rID)
+	fmt.Printf("%+v\n", header)
+	fmt.Printf("%+v\n", trailer)
+
```

Comment:
This obviously needs to be reverted, but I'm wondering what would be the right way to output the request id in the CLI.